consensus/module.go

```diff
@@ -1,18 +1,17 @@
-package consensus
+package  consensus
```

Comment:
Let me know if you need help integrating this with the actual consensus module so we can get it working end to end with LocalNet.